deployment/docker-compose.yml

```diff
@@ -55,7 +55,8 @@ web:
   # Note you cannot scale if you use conteiner_name
   extends: uwsgi-base
   container_name: channel-api-web
-  command: python manage.py runserver 0.0.0.0:8080
+  # using daphne server for running channel.
+  command: daphne -b 0.0.0.0 -p 8080 <app>.asgi:channel_layer
```

Comment:
please bake this into the Docker file and remove it from the compose file.

Reply:
Alright @cchristelis let me do so.